curp-external-api/src/cmd.rs

```diff
@@ -1,9 +1,10 @@
-use std::{fmt::Display, hash::Hash};
+use std::hash::Hash;
 
 use async_trait::async_trait;
 use engine::Snapshot;
 use prost::DecodeError;
-use serde::{de::DeserializeOwned, Deserialize, Serialize};
+use serde::{de::DeserializeOwned, Serialize};
+use uuid::Uuid;
```

Comment:
`curp` and `xline-client` no longer need to depend on `uuid`

Reply:
Removed